deploy/webhook.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
I don't think this is required, as it will be created automatically.  Also, I think whatever is specified here will be overridden by the webhooks registered through the code.

Reply:
no more self registration supported in v2